exercises/practice/largest-series-product/.docs/instructions.md

```diff
@@ -0,0 +1,14 @@
+# Instructions
+
+Given a string of digits, calculate the largest product for a contiguous substring of digits of length n.
+
+For example, for the input `'1027839564'`, the largest product for a series of 3 digits is 270 `(9 * 5 * 6)`, and the largest product for a series of 5 digits is 7560 `(7 * 8 * 3 * 9 * 5)`.
+
+Note that these series are only required to occupy *adjacent positions* in the input; the digits need not be *numerically consecutive*.
+
+For the input `'73167176531330624919225119674426574742355349194934'`,
```

Comment:
I know this is probably due to prob-specs formatting, but perhaps we should drop the single quotes to prevent any misconceptions about the type of the inputs.

Reply:
Yeah, this is coming from prob-specs. The single quotes will be there to show it is a string I assume, but maybe we should use double quotes?

Reply:
Double quotes are more stringy, I think. In Julia and Elixir at least, double quotes are for strings and single quotes are for something else (characters in Julia, charlists in Elixir). I can't think of any languages that allow single quotes for strings and not double quotes.

Reply:
I've opened a PR: https://github.com/exercism/problem-specifications/pull/2233